app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Response;
+use Illuminate\Support\Facades\Storage;
+use Mpdf\Mpdf;
+
+class UserController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        $users = User::all();
+        return view('dashboard.user', ['users' => $users]);
+    }
+
+
+    public function draw(Request $request)
+    {
+        $search = $request->query('search')['value'];
+        $draw = $request->query('draw', 1);
+        $start = $request->query('start', 0);
+        $length = $request->query('length', 10);
+        $usersQuery = User::query();
+        $totalusers = $usersQuery->count();
+
+        if (!$request->dropdowns) {
+            $users = $usersQuery->where('name', 'like', "%" . $search . "%")
+                ->orWhere('email', 'like', "%" . $search . "%")
+                ->orWhere('is_active', 'like', "%" . $search . "%");
+        } else {
+            $searchUser = $request->dropdowns['user'] ?? null;
+            $users = $usersQuery->whereIn('user', $searchUser);
+        }
+
+        if ($order = $request->query('order')[0] ?? null) {
+            $orderDir = $order['dir'] ?? 'asc';
+            $users->orderBy($order['name'], $orderDir);
+        }
+
+        $filteredusers = $search ? $users->count() : $totalusers;
+        $users = $users->skip($start)
+            ->take($length)
+            ->get();
+
+        return Response::json([
+            'draw' => intval($draw),
+            'recordsTotal' => intval($totalusers),
+            'recordsFiltered' => $filteredusers,
+            'data' => $users
+        ]);
+    }
+
+    public function createPDF(){
+        $document = new Mpdf();
+        $users = User::all();
+
+        $rows = '';
+        foreach ($users as $user) {
```

Comment:
create a blade file instead of using html tags into php code

Reply:
implemented above please review again #125